### PR TITLE
Allow call expressions in babel plugin

### LINKED ISSUE #33: Allow wrapping rpc function
Currently having the following code throws the error `rpc exports must be static functions` in the babel transform

```ts
export const config = { rpc: true }; // enable rpc on this API route


export const getName = wrapFunction(async function getName(code) {
  // ...
})
```

Wrapping the rpc function is useful to add error handling logic and tracing, the babel plugin should not error if the rpc function is a function wrapped by a function call

I can make a PR if you agree

### PR DESCRIPTION
**Pull request by remorses**
Fix #33

### CODE REVIEW

File: test/utils.js [hunk 1]
```diff
@@ -63,7 +63,9 @@ async function startNext(appPath, port) {
  * @returns Promise<void>
  */
 async function cleanup(appPath) {
-  await fs.rm(path.resolve(appPath, './.next'), { recursive: true });
+  try {
+    await fs.rm(path.resolve(appPath, './.next'), { recursive: true });
+  } catch {}
```
Comment (Janpot) [inline test/utils.js:68]: Which error is expected to be thrown here?

Comment (remorses) [reply test/utils.js:68]: If you run the tests the first time (no .next directory) it will throw `.next directory does not exist`

Comment (Janpot) [reply test/utils.js:68]: then we need to catch it as such

```suggestion
  } catch (err) {
    if (err.code !== 'ENOENT') {
      throw err;
    }
  }
```

**Comment by Janpot** [comment]
How should the behaviour be at runtime if someone does

```ts
export const notMethod = wrap('hello')
// or 
export const syncMethod = wrap(() => 'hello')
```
?

Is there a way to make it so the application fails to start up?

**Comment by remorses** [comment]
I don't think it's a good idea to prevent `wrap('hello')` because wrap could actually return a function even if the argument is not

Maybe if you really want to have this static check we could add some code at the end of the file that makes sure all the exports are function at runtime

```ts
export const notMethod = wrap('hello')

// generated code
const __exports = [{fn: notMethod, name: 'notMethod'}]

for (let { fn, name } of __exports) {
  if (typeof fn !== 'function') {
    throw new Error(`${name} is not a function`)
  }
}
```

This code would fail at build time because nextjs tries to prerender the app at build time

**Comment by Janpot** [comment]
Added in v3.4.0

**Pull request closed without merging**